lib/hyrax/configuration.rb

```diff
@@ -282,7 +282,7 @@ def bagit_dir
     # Path on the local file system where derivatives will be stored
     attr_writer :derivatives_path
     def derivatives_path
-      @derivatives_path ||= ENV.fetch('HYRAX_DERIVATIVE_PATH', Rails.root.join('tmp', 'derivatives'))
+      @derivatives_path ||= ENV.fetch('HYRAX_DERIVATIVES_PATH', Rails.root.join('tmp', 'derivatives'))
```

Comment:
Is this ENV variable name change a breaking change?

Reply:
the old env was introduced earlier this week. :)